plugin-api.d.ts

```diff
@@ -1227,14 +1227,14 @@ interface CodeBlockNode extends OpaqueNodeMixin, MinimalBlendMixin {
   clone(): CodeBlockNode
 }
 
-interface LayerSublayerNode {
+interface LabelSublayerNode {
   fills: Paint[] | PluginAPI['mixed']
 }
```

Comment:
I think this type is wrong! it should be closer to what the docs say https://www.figma.com/plugin-docs/api/LabelSublayer/